icu4j/tools/build/src/main/java/com/ibm/icu/dev/tool/errorprone/ErrorProneReport.java

```diff
@@ -266,4 +268,107 @@ private static void outSummary(HtmlUtils hu, Map<String, List<ErrorProneEntry>>
             hu.closeTag("p").nl();
         }
     }
+
+    private static void genReportTsv(Map<String, List<ErrorProneEntry>> errors,
+            String outDir) throws IOException {
+        Path outFileName = Paths.get(outDir, TSV_REPORT_FILE);
+        System.out.println("Report generated: " + outFileName);
+        try (PrintStream wrt = new PrintStream(outFileName.toString(), StandardCharsets.UTF_8)) {
+            wrt.println("Issue type"
+                    + "\tCount"
+                    + "\tSeverity"
+                    + "\tURL"
+            );
+            for (Map.Entry<String, List<ErrorProneEntry>> e : errors.entrySet()) {
+                String errorType = e.getKey();
+                List<ErrorProneEntry> errorsOfType = e.getValue();
+                if (errorsOfType.isEmpty()) {
+                    continue;
+                }
```

Comment:
optional: each of the genReport... methods is doing some of the same stuff. could we simplify? for example, filter empty entries from the input so that the methods need not check for empty?

Reply:
BTW, if it helps here are the reports:
* https://mihai-nita.net/tmp/ep/errorprone1.html
* https://mihai-nita.net/tmp/ep/errorprone2.html
* https://mihai-nita.net/tmp/ep/errorprone.md
* https://mihai-nita.net/tmp/ep/errorprone.tsv

Reply:
> The `isEmpty` part was just excess of care :-) It should never happen.
> 
> I removed, since in the end this is not production code. Even if it happens (how?) probably the fix should be somewhere else. Let's fail, and fix it if it does.

If you did remove this check, then I guess you have a pending change on your local disk which didn't make it into the main merge.

> In general the 4 reports are not that similar. There is not much to refactor.

ok